src/main/java/com/estimator/config/SecurityConfig.java

```diff
@@ -26,6 +27,9 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
     @Autowired
     private UserRepository userRepository;
 
+    @Autowired
+    private JwtTokenProvider jwtTokenProvider;
+
     @Bean
     public OAuth2UserService<OAuth2UserRequest, OAuth2User> customOAuth2UserService() {
```

Comment:
Duplicate imports are present in the code (for example, UserRepository, CustomOAuth2UserService and UserDetailsServiceImpl). This may be the result of automatic addition of dependencies or other changes in the file. Check and remove unnecessary imports to maintain code cleanliness.